spec/jobs/socure_shadow_mode_proofing_job_spec.rb

```diff
@@ -171,7 +170,19 @@
       }
     end
 
+    let(:known_reason_codes) do
+      {
+        I123: 'Person is over seven feet tall.',
+        R567: 'Person may be an armadillo.',
+        R890: 'Help! I am trapped in a reason code factory!',
```

Comment:
happens to me all the time

Reply:
Please update the DIVR to indicate how often armadillos over 7' tall are committing fraud from inside the reason code factory.